vars.mk

```diff
@@ -25,7 +25,7 @@ else
 endif
 
 ifeq ($(TAG),)
-  TAG := $(shell git describe --always --dirty 2> $(NULL))
+  TAG := $(shell git describe --always --dirty --abbrev=10 2> $(NULL))
```

Comment:
Interested as to why we changed this? Was the default too short?

Reply:
Too short or too long, depending the git version/os on the CI node...